File: 1863/1863.py
class Solution:
    def sum_of_all_subset_xor_totals(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.sum_of_all_subset_xor_totals(root.left), self.sum_of_all_subset_xor_totals(root.right))

File: 1863/1863.go
func sumOfAllSubsetXorTotals(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := sumOfAllSubsetXorTotals(root.Left)
	r := sumOfAllSubsetXorTotals(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}